skins/laika/src/components/pages/donation_confirmation/MembershipInfo.vue

```diff
@@ -15,13 +27,29 @@ import Vue from 'vue';
 
 export default Vue.extend( {
 	name: 'MembershipInfo',
+	props: [
+		'confirmationData',
+	],
+	computed: {
+		membershipApplicationUrl(): string {
+			const donation = this.confirmationData.donation;
+			return `apply-for-membership?donationId=${donation.id}&donationAccessToken=${donation.accessToken}&type=sustaining`;
+		},
+	},
 } );
 </script>
 
 <style lang="scss">
     @import "../../../scss/custom";
-    .membership-benefits-wrapper {
-        border: 1px solid $fun-color-primary-light;
-        border-radius: 2px;
+    .membership-benefits {
+        li {
+            &:before {
+                content: "– ";
```

Comment:
Styling lists in a typographically correct way (bullets outside of the line break area) should be done with `list-style`. Please confer with UX if bullets instead of dashes are ok. If yes, then use `list-style: disc`, otherwise you need [add margin trickery](https://stackoverflow.com/a/16507041/130121) since `list-style: "–"` is not  [yet supported on most browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type#Browser_compatibility).

Reply:
I checked with Jan and he said it's fine to use standard bullet points.